lib/cfg-tree.c

```diff
@@ -1074,7 +1075,7 @@ _verify_unique_persist_names_among_pipes(const GPtrArray *initialized_pipes)
 
       if (current_pipe_name != NULL)
         {
-          if (g_hash_table_contains(pipe_persist_names, current_pipe_name))
+          if (g_hash_table_lookup_extended(pipe_persist_names, current_pipe_name, &orig_key, &value))
```

Comment:
The last two parameters can be NULL, if we don't need them in the future (they hid it well in their [documentation](https://developer.gnome.org/glib/stable/glib-Hash-Tables.html#g-hash-table-lookup-extended): orig_key, value - [optional][nullable]).